examples/application/src/mutable_state.rs

```diff
@@ -1,6 +1,6 @@
 // <setup_mutable>
 use ntex::web;
-use std::sync::Mutex;
+use std::sync::{Arc, Mutex};
 
 struct AppStateWithCounter {
     counter: Mutex<i32>, // <- Mutex is necessary to mutate safely across threads
```

Comment:
You have to use Arc<Mutex<_>> here, or Arc<AppStateWithCounter> for `index` handler

Reply:
My bad i fixed it, and let's see if i fixed the ci build aswell